code/controllers/subsystem/fastprocess.dm

```diff
@@ -0,0 +1,36 @@
+var/datum/subsystem/fastprocess/SSfastprocess
+
+/datum/subsystem/fastprocess
+	name = "Fast Process"
+	priority = -1
+	wait = 1
+	dynamic_wait = 1
```

Comment:
Yes, in this case dwait will mainly just mean that the subsystem will not make up missed ticks, if it fires 2ds late, its next tick will still be 5ds away, not 3ds.